test/cases/bootstrapping.test.ts

```diff
@@ -1,51 +1,57 @@
-import basic from '../../sample-toplogies/basic.json';
-import advanced from '../../sample-toplogies/advanced.json';
-import {SetupJson} from '../../src/types/setup';
-import setupTestEnvironment from '../../src';
-import {LuminoTestEnvironment} from '../../src/types/lumino-test-environment';
-import {LuminoNode} from '../../src/types/node';
-import {isEmpty} from "../utils";
+import basic from "../../topologies/basic.json";
+import advanced from "../../topologies/advanced.json";
+import { SetupJson } from "../../src/types/setup";
+import setupTestEnvironment from "../../src";
+import { LuminoTestEnvironment } from "../../src/types/lumino-test-environment";
+import { LuminoNode } from "../../src/types/node";
+import { isEmpty } from "../utils";
 
 const TIMEOUT = 10 * 60 * 1000;
 
 describe("Bootstrapping Basic", () => {
-
-    let basicEnvironment: LuminoTestEnvironment;
-
-    it('should have basic nodes ready', async () => {
-        basicEnvironment = await setupTestEnvironment(basic as SetupJson);
-        expect(Object.keys(basicEnvironment.nodes).length).toBe(2);
-        const node0 = basicEnvironment.nodes.node0 as LuminoNode;
-        const node1 = basicEnvironment.nodes.node1 as LuminoNode;
-        const mauriAddress = (await node0.client.sdk.getAddress()).our_address;
-        const jonaAddress = (await node1.client.sdk.getAddress()).our_address;
-        expect(isEmpty(mauriAddress)).toBe(false);
-        expect(isEmpty(jonaAddress)).toBe(false);
-    }, TIMEOUT);
-
-    afterAll(async () => {
-       await basicEnvironment.stop();
-    }, TIMEOUT);
-
-})
+  let basicEnvironment: LuminoTestEnvironment;
```

Comment:
looks like some automatic formatting changes took place here

Reply:
mmm husky?

Reply:
yup when you push it changes your code, anyway looks good to me, except i don't like the quotes way, i always use a simple quote way for strings

Reply:
ftr this was done by the widely used [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension.

i suggest we leave it as-is and then settle on js/ts code practices at some point.

unless of course someone has an objection (i'm willing to revert these, it's really no problem)

Reply:
Yes no problem, we should change the rules later if we don't want to use quotes or things like that.